### PR TITLE
Cypress Tests: Progress Tracker

### PR DESCRIPTION
**Pull request by Abhishek-kumar09**
Signed-off-by: Abhishek kr <abhimait1909@gmail.com>

**Description**
- [x]  Load Pattern by click and verify rendering
- [x] Load All the visible components and checks RJSF rendering issues if any
This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: cypress/integration/e2e/login_spec.js [hunk 1]
```diff
@@ -49,8 +49,8 @@ describe("Login", () => {
     cy.contains("Filters")
   });
 
-  // visualizer test is skipped until the gql-plugin error sorts out in Meeshery
-  it.skip("Visit MeshMap Visualizer", () => {
+  // visualizer test is skipped until the gql-plugin error sorts out in meshery
+  it.skip("Visit MeshMap Visualiser", () => {
```
Comment (leecalcote) [inline cypress/integration/e2e/login_spec.js:53]: eeek. A step backwards here.

Comment (leecalcote) [reply cypress/integration/e2e/login_spec.js:53]: American English
<strikethrough>British English</strikethrough>

Comment (leecalcote) [reply cypress/integration/e2e/login_spec.js:53]: Those blokes and their "s"es. 🙄

**Comment by Abhishek-kumar09** [comment]
Got to know about problem with Linkerd's Mesh TLS component due to cypress test😎
[Screen Recording 2022-08-11 at 12.54.40 AM.zip](https://github.com/layer5labs/meshery-extensions-packages/files/9303739/Screen.Recording.2022-08-11.at.12.54.40.AM.zip)

**Pull request merged**